Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,54 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+# Do you like nature?
+default persistent._mas_pm_likes_nature = None
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_nature",category=['nature', 'you'],prompt="The outdoors",random=True))
+ 
+label monika_nature:
+    m 2esd "Hey, [player]..."
+    $ _history_list.pop()
+    menu:
+        m "Do you like nature{fast}?"
+        "Yes.":
+            $ persistent._mas_pm_likes_nature = True
+            m 2sub "Really? That's wonderful!"
```

Comment:
This line makes her seem kind of surprised at us for liking nature. I think a simple 1hua could work here instead, what do you think? Also, if we change the sentence to "That's wonderful!" instead, is my suggestion.

![1](https://user-images.githubusercontent.com/45746485/93427960-f79da580-f8be-11ea-9e56-454ec095c884.png)

Reply:
I think that liking nature is becoming more and more niche as time progresses (especially when it comes to stuff like camping, mountain climbing etc.) so I think it's fair that she'd be surprised, especially since we spend a lot of time indoors just to talk with Monika in the first place.

Reply:
"Really? That's wonderful!" sounds like she was half expecting us not to like it, instead of being neutral or not expecting anything either way IMO. Like it's a fine sentence to use for a question like "Do you like skydiving?" since it's a very niche hobby, while "Do you like nature?" is way more common. Liking nature is pretty much on par with liking exercise IMO, it shouldn't be something you go out of your way to like at least. That's my opinion, though I understand that people living in cities don't often go out into the forests and hills in certain parts of the world.

Reply:
Perhaps it's less surprise and more excitement. Monika doesn't really know what to expect from the player, but we can see through several topics that she enjoys nature and would probably be ecstatic if the player did as well. I don't see this as a scenario for hua IMO because she wouldn't just be happy, she'd be excited—hua gives me a 'calm happy' vibe, whereas sub seems more 'excited happy'. One solution here could be to compromise and change it to sua so that we keep the sparkle eyes but adjust her smile to be a bit calmer.